apps/client/src/app/publish/publish.component.ts

```diff
@@ -1,5 +1,5 @@
 import {Component, ElementRef, OnInit, ViewChild} from '@angular/core';
-import {ImplementationGuide} from '../../../../../libs/tof-lib/src/lib/stu3/fhir';
+import {ImplementationGuide} from '../../../../../libs/tof-lib/src/lib/r4/fhir';
```

Comment:
Can `IImplementationGuide` from `fhirInterfaces.ts` be used here instead of a version-specific instance of ImplementationGuide?